dev-tools/mage/goget.go

```diff
@@ -0,0 +1,39 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package mage
+
+import (
+	"path/filepath"
+
+	"github.com/magefile/mage/sh"
+	"github.com/pkg/errors"
+)
+
+// GoGet installs a go dependency from the vendor path
+func GoGet(importPath string) error {
+	beatsDir, err := ElasticBeatsDir()
+	if err != nil {
+		return errors.Wrap(err, "failed to obtain beats repository path")
+	}
+
+	if err := sh.Run("go", "get", filepath.Join(beatsDir, "vendor", importPath)); err != nil {
+		return errors.Wrap(err, "go get failed")
+	}
+
+	return nil
+}
```

Comment:
how about collecting go tooling into a separate package.

For example: https://github.com/elastic/go-txfile/blob/master/dev-tools/lib/mage/gotool/go.go

Samples how it's used:
- https://github.com/elastic/go-txfile/blob/master/magefile.go#L129
- https://github.com/elastic/go-txfile/blob/master/magefile.go#L156

Reply:
It would be better, yes. But I guess this is out of the scope of this PR?

We'd have to discuss where this package lives so it can be reused by the different projects.

Reply:
I'm fine with having it in the beats repo only for now.